static/app/components/alerts/snoozeAlert.tsx

```diff
@@ -34,30 +40,40 @@ function SnoozeAlert({isSnoozed, onSnooze, projectSlug, ruleId}: Props) {
           },
         }
       );
+
+      setDisabled(false);
+      onSnooze({
+        snooze: !isSnoozed,
+        snoozeCreatedBy: 'You',
+        snoozeForEveryone: target === 'me' ? false : true,
+      });
     } catch (err) {
+      if (err.status === 403) {
```

Comment:
Much better! But we also want to show an error message for errors that aren't 403.